processing file asmFiles/count_days.asm (1 of 12)
processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/test.ldst1.asm (2 of 5)
processing file asmFiles/test.ldst2.asm (3 of 5)
processing file asmFiles/test.loadstore.asm (4 of 5)
processing file asmFiles/test.rtype.asm (5 of 5)
--- memsim.hex	2015-02-20 10:12:53.157115374 -0500
+++ memcpu.hex	2015-02-20 10:12:53.140114894 -0500
@@ -35,5 +35,4 @@
 :04002700000021565E
 :04002800000D269011
 :04002900000006933A
-:04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/test.ldst1.asm (2 of 5)
processing file asmFiles/test.ldst2.asm (3 of 5)
processing file asmFiles/test.loadstore.asm (4 of 5)
processing file asmFiles/test.rtype.asm (5 of 5)
--- memsim.hex	2015-02-20 10:28:16.090148031 -0500
+++ memcpu.hex	2015-02-20 10:28:16.074147581 -0500
@@ -35,5 +35,4 @@
 :04002700000021565E
 :04002800000D269011
 :04002900000006933A
-:04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/test.ldst1.asm (2 of 5)
processing file asmFiles/test.ldst2.asm (3 of 5)
processing file asmFiles/test.loadstore.asm (4 of 5)
processing file asmFiles/test.rtype.asm (5 of 5)
--- memsim.hex	2015-02-20 10:29:19.970948241 -0500
+++ memcpu.hex	2015-02-20 10:29:19.954947789 -0500
@@ -35,5 +35,4 @@
 :04002700000021565E
 :04002800000D269011
 :04002900000006933A
-:04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/test.ldst1.asm (2 of 5)
processing file asmFiles/test.ldst2.asm (3 of 5)
processing file asmFiles/test.loadstore.asm (4 of 5)
processing file asmFiles/test.rtype.asm (5 of 5)
processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/test.ldst1.asm (2 of 5)
processing file asmFiles/test.ldst2.asm (3 of 5)
processing file asmFiles/test.loadstore.asm (4 of 5)
processing file asmFiles/test.rtype.asm (5 of 5)
